spec/environment-spec.js

```diff
@@ -0,0 +1,161 @@
+'use babel'
+/* eslint-env jasmine */
+
+import child_process from 'child_process'
+import environment from '../src/environment'
+import os from 'os'
+import _ from 'underscore-plus'
+
+describe('Environment handling', () => {
+  let originalEnv
+  let options
+
+  beforeEach(() => {
+    originalEnv = process.env
+    delete process._originalEnv
+    options = {
+      platform: process.platform,
+      env: _.clone(process.env)
```

Comment:
I'll try it out. I haven't used it before. If it works the same, I'll update it.